functions/package.json

```diff
@@ -1,33 +1,35 @@
 {
   "name": "functions",
-  "description": "Cloud Functions for Firebase",
   "scripts": {
-    "lint": "eslint .",
-    "serve": "firebase serve --only functions",
-    "shell": "firebase functions:shell",
+    "lint": "tslint --project tsconfig.json",
+    "build": "tsc",
+    "serve": "npm run build && firebase emulators:start --only functions",
+    "shell": "npm run build && firebase functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
-    "logs": "firebase functions:log",
-    "setup-firebase": "node setupProject.js"
+    "logs": "firebase functions:log"
   },
   "engines": {
     "node": "10"
   },
+  "main": "lib/index.js",
   "dependencies": {
+    "firebase-admin": "^8.10.0",
+    "firebase-functions": "^3.6.1",
```

Comment:
We can leave the other versions of firebase-admin and firebase-functions

Reply:
Sure!